source/releasenotes/2024-05-10-sendgrid-transactional-emails.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Transactional Emails: Rest API emails now sent via SendGrid effective May 12, 2024"
```

Comment:
```suggestion
title: "Transactional Emails: Rest API emails now sent via SendGrid effective May 12, 2024"
```

This wording is confusing @tyherbert. Are the emails "REST API emails" or are they just emails from WordPress and Drupal that will now be delivered by SendGrid instead of the previous method (which I think simply relied on the capacity of any server running to a CMS to try to send an email) and the mechanism of communication between our PHP runtime and SendGrid is SendGrid's REST API?

Reply:
I'm also not sure the word "transactional" is helping here. It might risk adding confusion. If we just mean "Emails sent by WordPress or Drupal" then let's say something plainer.

Reply:
I agree with you about the confusion. In this case, the term transactional email is and email type (generally, you see transactional vs marketing emails). They are categorized differently by email providers (marketing emails have different standards, opt-in, etc). If we call out transactional emails once (maybe in the title), we can remove this from the other areas.

Reply:
How does this look?
```suggestion
title: "Transactional Emails: Now sent via SendGrid effective May 12, 2024"
```